ghost/core/core/server/data/migrations/versions/5.111/2025-03-02-11-00-00-update-comments-permissions.js

```
@@ -0,0 +1,39 @@
+    addPermissionToRole({
+        permission: 'Read comments',
+        role: 'Editor'
+    }),
+    addPermissionToRole({
+        permission: 'Browse comments',
+        role: 'Editor'
+    }),
+    addPermissionToRole({
+        permission: 'Report comments',
+        role: 'Editor'
+    }),
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)